app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,13 @@
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :firstName, String, null: true
+    field :lastName, String, null: true
+    field :email, String, null: true
+    field :password, String, null: true
+    field :speed, Integer, null: true
```

Comment:
speed and pitch are what we had listed in out notes, might have to change those depending on what the api needs. that would be an easy fix.